### DISCUSSION TITLE
Cannot switch between pages

### DISCUSSION THREAD

**Discussion by jaltekruse** [edemaine/cocreate#211] [Q&A] [answered]
I tried running a server locally using the tip of master and I am not able to move between different pages on a board. In this video clicking the page tab and the let right arrows doesn't appear to change anything. I also tried grab the link for navigating to the second page, which has a reference to an element ID and open it in a new tab, but on page load the ID is stripped off the URL and it just shows the first page.

https://user-images.githubusercontent.com/365366/177906542-8992e12a-93f2-4e75-b97f-7118ddfcf7c8.mp4

**Comment by edemaine** [comment]
Did you check the JavaScript console, and the console where the server is running? I imagine there's an error in one of those places that would help debug. (I've never seen this behavior.)

**Comment by jaltekruse** [comment]
There are a few warnings in the terminal from the startup of meteor, but I don't see any new logs there as I click on the tabs, or any reported errors in the JS console in the browser.

```
jason@pew-pew-puter:~/src/cocreate$ meteor
[[[[[ ~/src/cocreate ]]]]]                    

=> Started proxy.                             
=> Started MongoDB.                           
(node:2753966) Warning: Accessing non-existent property 'lineno' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
(node:2753966) Warning: Accessing non-existent property 'column' of module exports inside circular dependency
(node:2753966) Warning: Accessing non-existent property 'filename' of module exports inside circular dependency
(node:2753966) Warning: Accessing non-existent property 'lineno' of module exports inside circular dependency
(node:2753966) Warning: Accessing non-existent property 'column' of module exports inside circular dependency
(node:2753966) Warning: Accessing non-existent property 'filename' of module exports inside circular dependency
I20220707-12:58:00.201(-5)? Upgraded database as necessary.
I20220707-12:58:00.263(-5)? Monti APM: completed instrumenting the app
=> Started your app.

=> App running at: http://localhost:3000/
=> Meteor 2.7.3 is available. Check the changelog https://docs.meteor.com/changelog.html and update this project
with 'meteor update'.
```

https://user-images.githubusercontent.com/365366/178005910-0de0b5a1-dc66-4ad4-9972-149c0b6d3c3c.mp4

**Reply by edemaine** [reply]
Yeah, those server warnings are all normal.

I can't see the with-console video. Can you try again?

Just checking: Did you `npm install`?

**Reply by jaltekruse** [reply]
Sorry forgot to finalize the recording, so apparently that gives me something that looks like a video file but is unplayable.

I was following the install instructions so I ran `meteor npm install`


https://user-images.githubusercontent.com/365366/178012579-1171e7a7-b787-487a-9f12-ad97c629f3d8.mp4

**Comment by edemaine** [comment, marked as answer]
It turns out that this was a dependency issue. In the latest commit, I've pinned all dependency versions to prevent this from happening in the future. Also, I explicitly kept `solid-app-router` at an older version, which was the source of the problem. I'll work on upgrading meanwhile, but at least a fresh install should work again now.

**Reply by jaltekruse** [reply]
Thanks for the quick response! I can confirm the latest commit does fix the issue.
